source/administration/upgrade.md

```diff
@@ -11,19 +11,16 @@ To start, select one of the following guides:
 
 ## Upgrade Team Edition to 3.1.x and later
 
-**Important Note:** Security-related changes were made in 3.8 that require you to verify settings in the System Console before upgrading from version 3.7.x and earlier to any version greater than 3.8.0.
-
-**To prepare your system when upgrading from 3.7.x and earlier**:
-
-  1. In the System Console, go to **General > Configuration** and make sure that the **Site URL** is specified. It must not be empty.
-  2. In the System Console, go to **General > Logging** and make sure that the **File Log Directory** field is either empty or has a directory path only. It must not have a filename as part of the path.
+**Important Note:** Security-related changes were made in `v3.6.x` and later that require you to specify the Site URL setting in the System Console before upgrading from version `v3.5.x` and earlier to any version greater than `v3.6.x`. In the System Console, go to **General > Configuration** and make sure that the **Site URL** is specified. It must not be empty.
```

Comment:
But someone upgrading from 3.7.1 to 3.8.2 (for instance) will need to add it too?